docs/team/member-guide.md

```diff
@@ -98,4 +101,56 @@ deciding to merge things into one of our repositories:
   some time, but for most changes it is fine to just go ahead and merge.
   Again, we trust your judgment, and we don't want these guidelines to become
   a burden that slows down development.
-  
+
+## Periodic team check-ins
+
+We want team membership to be an opportunity to grow and have an impact, not
+a burden or a time sink. While we ask for a minimum 6 month commitment when
+you join a JupyterHub or Binder team, we don't expect this commitment to
+last forever.
+
+In order to ensure that team membership is still the best option for you,
```

Comment:
The way this is phrased somehow makes me think of company announcements where they increase prices, reduce service and tell me that this is good for me. How about starting directly with "We try to conduct..."?